troubleshooting/incrementally_configuring_images.md

```diff
@@ -0,0 +1,297 @@
+# Incrementally configuring images
+
+When IUF or SAT `bootprep` are failing to configure an image, this procedure can be used to incrementally configure an image.
+This avoids re-running the build and configuration steps that have succeeded, which would occur if IUF or SAT `bootprep` were re-run.
+
+The following example assumes the user is creating a `Compute` image, but can be adapted for other image types.
+
+## Setting up
+
+1. (`ncn-mw#`) Find the path to the IUF logs printed by the `iuf` command. It will look like this:
+
+   ```text
+   2023-11-01T14:06:36.128255Z INFO All logs will be stored in /etc/cray/upgrade/csm/iuf/cos-products-20231031/log/20231101140636
+   ```
+
+   See `install.log` in this directory when subsequent steps reference the IUF logs.
+
+1. (`ncn-mw#`) Find the SAT logs by looking for a line like the following in the IUF logs:
+
+   ```text
+   2023-11-01T14:58:22.874482Z INFO [prepare-managed-images] END sat-bootprep-run [Failed]
+   2023-11-01T14:58:22.874567Z DBG  [prepare-managed-images] LOG FILE FOR sat-bootprep-run: argo/cos-products-20231031-gdm63-prepare-images-2rkds/cos-products-20231031-gdm63-prepare-images-2rkds-sat-wrapper-2664865581/main.log
+   ```
+
+   The string starting with `argo/` is the key within the `config-data` bucket of the S3 object
+   which contains the log output from `sat bootprep`. Save this S3 key to an environment variable:
+
+   ```bash
+   export LOGS_S3_KEY="argo/cos-products-20231031-gdm63-prepare-images-2rkds/cos-products-20231031-gdm63-prepare-images-2rkds-sat-wrapper-2664865581/main.log"
```

Comment:
It doesn't look like this procedure calls any scripts or commands which look for the presence of this environment variable, so we can just set it rather than export it. Exporting it is only needed when you will be starting up a child process and the environment variable needs to be set in that process.

```suggestion
   LOGS_S3_KEY="argo/cos-products-20231031-gdm63-prepare-images-2rkds/cos-products-20231031-gdm63-prepare-images-2rkds-sat-wrapper-2664865581/main.log"
```